runtime/ztests/expr/function/quiet.yaml

```diff
@@ -0,0 +1,19 @@
+zed: quiet(this)
```

Comment:
Nit: Merge this file with runtime/ztests/op/yield-quiet.yaml since they contain equivalent Zed programs.